VelocitySDK/Core/VLTApiClient.m

```diff
@@ -124,6 +125,8 @@ - (void)uploadMotionData:(nullable VLTPBCapture *)capture
 
     NSDictionary *params = @{
         LabelsKey: labels,
+        UserIDKey: [VLTUserDataStore shared].userId,
+        DataLengthKey: @(data.length)
```

Comment:
This seems redundant for me. Can you explain why we need to have this data length?

Reply:
Good point. Wonder why we put it in the card. It's actually safer to look at the data and figure out length.